ui/components/Connect/helpers/common.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Verify connection status
+ * If connection already exist response will be connection object
+ * If connection is not exist response will be 200 empty
+ * 
+ * @param {string} connectionKind - Kind of connection e.g (helm, github) pass only managed connections
+*/
+export const VerifyConnectionKind = async (connectionKind) => {
+    return await axios.post(`/api/integrations/${connectionKind}/status`);
```

Comment:
@aabidsofi19 are we getting rid of axios?

Reply:
yup , we are . @Yashsharma1911 you can create a rtk endpoint(s) for connections for now ( till we start using the codegen)

Reply:
Removed, RTK used